### ISSUE TITLE
Add user agent to outbound calls

### ISSUE THREAD

**Issue by tolzhabayev** [grafana/plugin-validator#46]
To eliminate false positives like this:
![SCR-20221228-gau](https://user-images.githubusercontent.com/580672/210218313-c7c92c1c-d3f2-4e0a-af1b-447a8e465ce5.png)

we could add a user agent to outbound calls.

Here is an example of cloudflare rejecting calls without a user agent.
```
curl -I https://support.anodot.com/hc/en-us/articles/360002631114-Token-Management-

HTTP/2 403 
date: Wed, 28 Dec 2022 10:49:31 GMT

curl -I -A "Grafana Plugin Validator 2.0" https://support.anodot.com/hc/en-us/articles/360002631114-Token-Management-
HTTP/2 200 
date: Wed, 28 Dec 2022 10:49:45 GMT
```

**Which user agent to use?**
To be defined by the one who picks up this issue, be aware that some default user agents e.g. chrome / firefox are also sometimes blocked as providers, for example cloudflare, detect somehow that you are using curl and not a real browser.

Might be just a case of replacing this string with whatever works best:
https://github.com/grafana/plugin-validator/blob/main/pkg/plugin/checkers.go#L277
